osu.Game/Screens/Edit/Screens/Compose/Timeline/BeatmapWaveformGraph.cs

```diff
@@ -11,14 +10,13 @@ namespace osu.Game.Screens.Edit.Screens.Compose.Timeline
 {
     public class BeatmapWaveformGraph : CompositeDrawable
     {
-        public readonly Bindable<WorkingBeatmap> Beatmap = new Bindable<WorkingBeatmap>();
+        public WorkingBeatmap Beatmap { set => graph.Waveform = value.Waveform; }
```

Comment:
Haven't seen this done anywhere else, are we fine with this being on one line?

Reply:
For simple statements, this is fine.